src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -818,6 +818,8 @@ public OptionsViewModel Options
 
         public string PackagePath => _searchResultPackage?.PackagePath;
 
+        public bool IsCentralPackageManagementEnabled { get; set; }
```

Comment:
Can we get rid of this property? Unless I missed a XAML binding, it's only used to store the result of IsCentralPackageManagementEnabledAsync. If we can avoid maintaining state, that's usually best. If we do want state, then I'm not sure where we're utilizing it?

Reply:
I'm using it to know if we should have floating versions enabled in the combobox https://github.com/NuGet/NuGet.Client/pull/5011/files#diff-0b6aa73227964074dc06409994723b4f7d158fe4edc669b5c8229c1c3d235bd4R342